src/components/inputWithLabel.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useRef } from "react"
+import style from "../TodoListItem.module.css"
+import PropTypes from "prop-types";
+
+function InputWithLabel({ children, name, id, value, onChange }) {
+    const inputRef = useRef()
+    useEffect(() => {
+        inputRef.current.focus()
+    })
+    return (
+        <>
+            <label htmlFor={id}>{children}:</label>
+            <input className={style.input} name={name} id={id} value={value} onChange={onChange} ref={inputRef} />
+        </>
+    )
+}
+
+InputWithLabel.propTypes = {
+    children: PropTypes.string.isRequired,
```

Comment:
Nitpick:
When it comes to typing `children` props, typically you'll use `PropTypes.node` since it covers a wide range of data types: boolean, string, number and even React components!